### PR TITLE
Fix: Styling not applying when used as a reusable block for Advanced Columns, Section & Marketing Button Blocks.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- trello - https://trello.com/c/pAHu2EUV/199-section-and-marketting-button-color-gradient-not-applying-when-used-as-a-reusable-block

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create a Section/Marketing button with color/gradient and save as a reusable in master.
- Update and refresh the page you will see the color style is not applying.
- Switch to `reusable-block-issue` branch refresh the editor and issue resolved.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/section/edit.js [hunk 1]
```diff
@@ -69,6 +69,12 @@ class UAGBSectionEdit extends Component {
 		const $style = document.createElement( "style" )
 		$style.setAttribute( "id", "uagb-section-style-" + this.props.clientId.substr( 0, 8 ) )
 		document.head.appendChild( $style )
+
+		var element = document.getElementById( "uagb-section-style-" + this.props.clientId.substr( 0, 8 ) )
```
Comment (RajkiranBagal) [inline src/blocks/section/edit.js:73]: Why again retrieve the element again, we already have it in $style variable.

File: readme.txt [hunk 2]
```diff
@@ -167,6 +167,9 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 
 == Changelog ==
 
+= 1.22.0 =
+* Fix: Styling not applying when used as a reusable block for section/marketting button.
```
Comment (RajkiranBagal) [inline readme.txt:171]: I also see a change in the Column block, please add also that in the changelog.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 2 above)
Comment (AbhijitNage123) [reply readme.txt]: Updated

File: src/blocks/section/edit.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/section/edit.js]: Updated

File: src/blocks/marketing-button/edit.js [hunk 3]
```diff
@@ -73,6 +73,12 @@ class UAGBMarketingButtonEdit extends Component {
 		$style.setAttribute( "id", "uagb-style-marketing-btn-" + this.props.clientId.substr( 0, 8 ) )
 		document.head.appendChild( $style )
 
+		var element = document.getElementById( "uagb-style-marketing-btn-" + this.props.clientId.substr( 0, 8 ) )
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/blocks/marketing-button/edit.js]: Updated

File: src/blocks/column/edit.js [hunk 4]
```diff
@@ -53,6 +53,12 @@ export default class UAGBColumnEdit extends Component {
 		const $style = document.createElement( "style" )
 		$style.setAttribute( "id", "uagb-column-style-" + this.props.clientId.substr( 0, 8 ) )
 		document.head.appendChild( $style )
+
+		var element = document.getElementById( "uagb-column-style-" + this.props.clientId.substr( 0, 8 ) )
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply src/blocks/column/edit.js]: Updated

File: languages/ultimate-addons-for-gutenberg-es_ES-a4f783c851c57db1967c3d9a58cfd198.json [hunk 5]
```diff
@@ -0,0 +1 @@
... activa"],"Tabs Body Settings":["Ajustes del cuerpo de las pesta\u00f1as"],"Body Background Color":["Color de fondo del cuerpo"],"Body Text Color":["Color del texto del cuerpo"],"Tab Body Margin (px)":["Margen del cuerpo de la leng\u00fceta (px)"],"Body Padding":["Acolchado corporal"],"Border Settings":["Configuraci\u00f3n de bordes"],"Border width":["Ancho del borde"],"Title\u2026":[""],"Remove tab":[""],"Add tab":[""],"lottie":[""],"animation":[""],"Controls":["Controla"],"Play On":["Jugar en"],"On Hover":["En hover"],"On Click":["Al hacer clic"],"Viewport":["Viewport"],"This setting will only take effect once you are on the live page, and not while you're editing.":["Esta configuraci\u00f3n solo surtir\u00e1 efecto una vez que est\u00e9 en la p\u00e1gina en vivo y no mientras est\u00e1 editando."],"Loop":["Bucle"],"Speed":["Velocidad"],"Reverse":["Marcha atr\u00e1s"],"Direction of animation.":["Direcci\u00f3n de animaci\u00f3n."],"Lottie block requires the file type JSON to be uploaded to media files. Seems like your website has disabled this file type. Please refer":[""],"Allows you to add fancy animation i.e Lottie to your website. You can see sample Lottie animations":[""]}}}
```
[the remark this answers is not in the document]
Comment (AbhijitNage123) [reply languages/ultimate-addons-for-gutenberg-es_ES-a4f783c851c57db1967c3d9a58cfd198.json]: Removed

File: src/blocks/marketing-button/edit.js [hunk 6]
```diff
@@ -73,6 +73,10 @@ class UAGBMarketingButtonEdit extends Component {
 		$style.setAttribute( "id", "uagb-style-marketing-btn-" + this.props.clientId.substr( 0, 8 ) )
 		document.head.appendChild( $style )
 
+		if( null !== $style && undefined !== $style ) {
```
Comment (RajkiranBagal) [inline src/blocks/marketing-button/edit.js:76]: Do we really need to check the null & undefined condition here, as we are creating the element here itself?

File: readme.txt [hunk 7]
```diff
@@ -167,6 +167,9 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 
 == Changelog ==
 
+= 1.22.0 =
+* Fix: Styling not applying when used as a reusable block for column/section/marketting button.
```
Comment (RajkiranBagal) [inline readme.txt:171]: Update changelog to `Fix: Styling not applying when used as a reusable block for Advanced Columns, Section & Marketing Button Blocks.
`

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 7 above)
Comment (AbhijitNage123) [reply readme.txt]: Updated

**Review by RajkiranBagal — APPROVED** [review]

**Pull request closed without merging**